v22.1/aws-dms.md

```diff
@@ -182,69 +155,60 @@ The `BatchApplyEnabled` setting can improve replication performance and is recom
 > SELECT table_catalog, table_schema, table_name, column_name FROM information_schema.columns WHERE is_hidden = 'YES';
 ~~~
 
-- **Drop tables on target** is currently not supported and will error on import.
+- **Drop tables on target** is not supported on v22.1 and earlier, and will error on import.
 
-{% comment %}
-- Not all schema objects are migrated when using **Drop tables on target** as a target table preparation mode. Thus, it is not supported at this time. A list of supported DDL statements within AWS DMS are mentioned [here](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Introduction.SupportedDDL.html). Any statements outside of that (creating or modifying foreign keys, secondary indexes, constraints, etc.) will have to be run manually in the target database.
-{% endcomment %}
+- On v22.1.0 to v22.1.6, a migration may fail if there is an odd number of `"` characters in a row. AWS DMS will return an error message like the following: `[TARGET_LOAD ]D: Command failed to load data with exit error code 0.`. This is resolved in [v22.1.7 and later](../releases/v22.1.html).
 
 ## Troubleshooting common issues
 
-Run the following query from within the target CockroachDB cluster to identify common problems with any tables that may be migrated:
+- For visibility into migration problems:
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-> WITH
-    invalid_columns
-        AS (
-            SELECT
-                'Table '
-                || table_schema
-                || '.'
-                || table_name
-                || ' has column '
-                || column_name
-                || ' which is hidden. Either drop the column or mark it as not hidden for DMS to work.'
-                    AS fix_me
-            FROM
-                information_schema.columns
-            WHERE
-                is_hidden = 'YES'
-                AND table_name NOT LIKE 'awsdms_%'
-        ),
-    invalid_version
-        AS (
-            SELECT
-                'This cluster is on a version of CockroachDB which does not support AWS DMS. CockroachDB v21.2.13+ or v22.1+ is required.'
-                    AS fix_me
-            WHERE
-                split_part(
-                    substr(
-                        substring(
-                            version(),
-                            e'v\\d+\\.\\d+.\\d+'
+    - Check the `SQL_EXEC` [logging channel](logging-overview.html#logging-channels) for log messages related to `COPY` statements and the tables you are migrating.
+    - Check the [Amazon CloudWatch logs that you configured](#step-2-2-task-settings) for messages containing `SQL_ERROR`.
+
+- Run the following query from within the target CockroachDB cluster to identify common problems with any tables that were migrated:
+
+    {% include_cached copy-clipboard.html %}
```

Comment:
This is a pretty large query here. Should this have any explanation on the SQL? There is a point about what it does, but I'm not sure how, or what to expect when/after it runs.